featurebyte/query_graph/sql/interpreter/preview.py

```diff
@@ -966,6 +981,7 @@ def construct_value_counts_sql(
         num_rows: int,
         num_categories_limit: int,
         seed: int = 1234,
+        convert_keys_to_string: bool = True,
```

Comment:
@gee-senbong The new behaviour will likely break things. I added this option to opt in to the new behaviour for smoother transition.